samples/cpp/benchmark_app/main.cpp

```diff
@@ -845,6 +848,8 @@ int main(int argc, char* argv[]) {
             auto compile_model_mem_start = get_peak_memory_usage();
             startTime = Time::now();
             compiledModel = core.compile_model(model, device_name, device_config);
+            // model is not required anymore
+            model.reset();
```

Comment:
It's a nice addition to to `read_model()` docstring. But at the same time RAII is the preferred way to organize code